### PR TITLE
Publish the audiofile type for UploadButton

### PR DESCRIPTION
**Pull request by MartaBerman**

### CODE REVIEW

File: js-wixcode-sdk/$w/UploadButton.service.json [hunk 1]
```diff
@@ -83,32 +84,33 @@
...
                 "",
-                " The file types of the images and documents that can be uploaded are",
+                " The file types that can be uploaded are",
                 " listed [here](https://support.wix.com/en/article/supported-media-file-types-and-file-sizes)." ],
             "links": [],
             "examples":
               [ { "title": "Get an upload button's type",
                   "body":
-                    [ "let fileType = $w(\"#myUploadButton\").fileType;  // \"Image\"" ],
+                    [ "let fileType = $w(\"#myUploadButton\").fileType;" ],
```
Comment (wixysam) [inline js-wixcode-sdk/$w/UploadButton.service.json:113]: Why did you remove the example value?

File: js-wixcode-sdk/$w/UploadButton.service.json [hunk 2]
```diff
@@ -419,8 +427,8 @@
         "url": "https://www.wix.com/code/home/forum/wix-tips-and-updates/example-wix-animations" } ],
   "FAQs":
     [ { "title": "What types of files can site visitors upload with an upload button?",
-        "markdown": "You can set your upload button to upload images, documents, or videos using the [`fileType`](https://www.wix.com/code/reference/$w.UploadButton.html#fileType) property. The file types of the images, documents, and videos that can be uploaded are listed [here](https://support.wix.com/en/article/supported-media-file-types-and-file-sizes)." },
+        "markdown": "You can set your upload button to upload images, documents, videos, or audios using the [`fileType`](https://www.wix.com/code/reference/$w.UploadButton.html#fileType) property. The file types of the images, documents, videos and audios that can be uploaded are listed [here](https://support.wix.com/en/article/supported-media-file-types-and-file-sizes)." },
```
Comment (wixysam) [inline js-wixcode-sdk/$w/UploadButton.service.json:430]: The word `audios` sounds weird to me here.

File: js-wixcode-sdk/$w/UploadButton.service.json (hunk 1 above)
Comment (MartaBerman) [reply js-wixcode-sdk/$w/UploadButton.service.json:113]: Because it looks like only Image is OK.  I liked it better when it listed the 4 types. But you didn't like that.

File: js-wixcode-sdk/$w/UploadButton.service.json (hunk 2 above)
Comment (MartaBerman) [reply js-wixcode-sdk/$w/UploadButton.service.json:430]: Yeah, I pondered that. Was trying it on for size. I didn't want to say "file" for each thing ("video file", "image file") I wonder why audios sounds bad if videos sounds OK.... but I agree it sounds a bit odd. 

I guess I have to now do another pull request in the original repo to make the change.
